src/QEQ/fix_qeq.h

```diff
@@ -70,10 +70,10 @@ class FixQEq : public Fix {
   int maxwarn;                 // print warning when max iterations was reached
   double cutoff, cutoff_sq;    // neighbor cutoff
 
-  double *chi, *eta, *gamma, *zeta, *zcore;    // qeq parameters
+  double *chi, *eta, *gamma, *zeta, *zcore, *qmin, *qmax, *omega;
   double *chizj;
   double **shld;
-  int streitz_flag, reax_flag;
+  int streitz_flag, reax_flag, ctip_flag;
```

Comment:
I think this `ctip_flag` flag is OK to keep in the parent class so the parsing doesn't need to be duplicated in the child class.

Reply:
exactly.